### PR TITLE
fix spark version in gdmix image and python module dependency 

### PR DESCRIPTION
**Pull request by catdollar**
- fix latest available spark 2 version in gdmix image Dockerfile
- The python wheel was published with wrong dependency, PyPi doesn't allow to overwrite old version, has to release a new version of 0.2.1

### CODE REVIEW

File: gdmix-workflow/images/gdmix/Dockerfile [hunk 1]
```diff
@@ -35,14 +35,14 @@ RUN rm -rf /usr/bin/python && \
...
-# Install spark 2.4.6 and its dependencies
+# Install spark 2.4.7 and its dependencies
 RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
     mkdir -p /opt/spark && \
     touch /opt/spark/RELEASE && \
     echo "auth required pam_wheel.so use_uid" >> /etc/pam.d/su && \
     chgrp root /etc/passwd && chmod ug+rw /etc/passwd
 
-ARG spark_version=2.4.6
+ARG spark_version=2.4.7
```
Comment (junshi15) [inline gdmix-workflow/images/gdmix/Dockerfile:45]: Why do we have to use 2.4.7 instead of 2.4.6?

**Review by junshi15 — COMMENTED** [review]
Original file, Line 15, 19:
sys.exit(1)
sys is not imported.

File: gdmix-workflow/images/gdmix/Dockerfile (hunk 1 above)
Comment (catdollar) [reply gdmix-workflow/images/gdmix/Dockerfile:45]: Because the 2.4.6 is not downloadable any more, 2.4.7 has been tested and is compatible.

**Comment by catdollar** [comment]
@junshi15  Fixed the sys import error, thanks!

**Review by junshi15 — APPROVED** [review]

**Pull request merged**